utils/testing.test.ts

```diff
@@ -13,3 +13,12 @@ describe("combineStrings", () => {
     expect(permutations).toEqual(expectedPermutations);
   });
 });
+
+describe("randomUnicodeString", () => {
+  it("should generate random unicode strings of the given length", () => {
+    for (let i = 0; i < 1024; i++) {
```

Comment:
1024 seems a little excessive, does it impact test performance in a significant way ? if so lets just pick 10 specific lengths and test them out instead of a full 1024 list

Reply:
Doesn't seem to impact performance at all. Runs about as fast as other unit tests.